src/Common/src/System/IO/PathInternal.Unix.cs

```diff
@@ -22,6 +22,8 @@ internal static bool HasIllegalCharacters(string path)
             Debug.Assert(path != null);
             return path.IndexOf(InvalidPathChar) >= 0;
         }
+        
+        internal static char[] GetInvalidPathChars() => new char[] { InvalidPathChar };
```

Comment:
Maybe this should be closer to the top of the file, under `InvalidPathChar` so it is near the other definitions.